.github/workflows/arm64-build-checks.yml

```diff
@@ -0,0 +1,47 @@
+name: arm64 build checks
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install dependency package
+      run: |
+        sudo sed -i -E 's|^deb ([^ ]+) (.*)$|deb [arch=amd64] \1 \2\ndeb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports/ \2|' /etc/apt/sources.list
+        sudo dpkg --add-architecture arm64
+        sudo apt-get update
+        sudo apt-get install -y --no-install-recommends \
+                crossbuild-essential-arm64 \
+                git \
+                cmake \
+                ca-certificates \
+                libpython-dev:arm64 \
+                libpython3-dev:arm64 \
+                libusb-1.0-0-dev:arm64 \
+                python-numpy \
+                python3-numpy \
+                libgstreamer1.0-dev:arm64 \
+                libgstreamer-plugins-base1.0-dev:arm64
+
+    - name: Fetch opencv_contrib
+      run: |
+        git clone --depth 1 https://github.com/opencv/opencv_contrib.git ../opencv_contrib
+
+    - name: Configure
+      run: |
+        mkdir build
+        cd build
+        cmake -DPYTHON_DEFAULT_EXECUTABLE=/usr/bin/python3 \
+              -DCMAKE_TOOLCHAIN_FILE=../platforms/linux/aarch64-gnu.toolchain.cmake \
+              -DOPENCV_EXTRA_MODULES_PATH=../../opencv_contrib/modules ../
```

Comment:
Can you please show CMake summary?

Reply:
I post it into a gist, see https://gist.github.com/Yikun/6bcb6080a0d8f542923ed637c7bbfb18 , or you could just see it in my local repo: https://github.com/Yikun/opencv/pull/2

Reply:
>  Unavailable:  python2 python3

Python bindings are disabled